eventq_rabbitmq/lib/eventq_rabbitmq/rabbitmq_queue_worker.rb

```diff
@@ -327,6 +324,11 @@ def configure(queue, options = {})
 
       private
 
+      def new_client_instance(options)
+        raise "[#{self.class}] - MQ Endpoint not present." unless options[:mq_endpoint]
+        EventQ::RabbitMq::QueueClient.new({endpoint: options[:mq_endpoint] })
```

Comment:
This is the queue client. Its a different animal. 

This is what we new up inside;

Confusing but a different set of options;

EventQ::RabbitMq::QueueClient.new({endpoint: options[:mq_endpoint] })